api/applications/applicationRouter.js

```diff
@@ -347,4 +349,28 @@ router.put(
   }
 );
 
+// posts mentee data to DS
+router.post('/dsMenteeData', authRequired, (req, next) => {
+  const formData = req.body;
+  axios
+    .post(`${process.env.DS_API_URL}`, formData)
+    .then((res) => {
+      Application.insertMenteeIntake(formData);
+      res.status(200).json(res);
+    })
+    .catch(next);
+});
+
+// posts mentor data to DS
+router.post('/dsMentorData', authRequired, (req, next) => {
+  const formData = req.body;
+  axios
+    .post(`${process.env.DS_API_URL}`, formData)
+    .then((res) => {
+      Application.insertMentorIntake(formData);
+      res.status(200).json(res);
+    })
+    .catch(next);
+});
```

Comment:
What is the intention of this route? When will these endpoints be called and by whom? Would interacting with the DS server make more sense in middleware?

Reply:
These routes are intended to receive an applicant's form data on clicking 'Submit' on the given form page, thus triggering a call on every registration attempt to send that form data both to our database _and_ DS at the same time.

That said, implementing this functionality in specialized middleware is an approach I hadn't yet considered - this follows the ways we've integrated WEB & DS in projects past, but middleware may be a more elegant approach to accomplish the same thing!

Either way, we'll still need a few more endpoints to fulfill the rest of our needs for DS. (Sending them a mentee ID and receiving suggested mentor matches in return, updating form data if the need arises, updating user profile info.)